test_scripts/API/SeparatingChangeAudibleStatus/common.lua

```diff
@@ -0,0 +1,183 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local test = require("user_modules/dummy_connecttest")
```

Comment:
Try to not use `test` module. E.g. `test.mobileSession` can be easily replaced by the function from `action` module